### ISSUE TITLE
Show schedule

### ISSUE THREAD

**Issue by jfly** [cubing/ccm#21]
(pulled this out of #6)

We need some way of asking organizers for start and end times for the rounds they plan to run.

(There are a lot of cool things we could do here. A drag and drop calendar ui would be nice. We could also estimate how realistic a schedule is as a function of the number of staff/stations, or even autogenerating 1 or several suggested schedules. All of that is out of scope for Phase 1)

**Comment by jfly** [comment]
@FatBoyXPC had a very clever idea for creating new events on the calendar. I'll leave it to him to either write it up here, or just secretly implement it =)

**Comment by FatBoyXPC** [comment]
I went ahead and implemented that idea in https://github.com/cubing/ccm/pull/160

**Comment by FatBoyXPC** [comment]
I don't think that the cutoff times should be on the schedule. This information can be found on the events page. While it'd obviously be handy, I think it will start to clutter things up.
